src/character_attire.cpp

```diff
@@ -1759,7 +1759,8 @@ void outfit::absorb_damage( Character &guy, damage_unit &elem, bodypart_id bp,
         if( outermost && elem.type == damage_type::HEAT && elem.amount >= 1.0f ) {
             // TODO: Different fire intensity values based on damage
             fire_data frd{ 2 };
-            destroy = !armor.has_flag( flag_INTEGRATED ) && armor.burn( frd );
+            destroy = !armor.has_flag( flag_INTEGRATED ) && !armor.has_flag( flag_UNBREAKABLE ) &&
+                      armor.burn( frd );
```

Comment:
This check likely shouldn't be here, if item::burn(...) below checks for UNBREAKABLE

Reply:
Yeah, thought having the flags together would make it easier to phase out INTEGRATED invulnerability in the future, but it is unnecessarily redundant. I'll get rid of it.